google-cloud-logging/docs/instrumentation.md

```diff
@@ -1,46 +1,136 @@
-# Stackdriver Logging Instrumentation
+# Stackdriver Debugger Instrumentation
```

Comment:
Did this file (`google-cloud-logging/docs/instrumentation.md`) get caught up in search & replace by accident?